Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -59,27 +59,37 @@ struct PublicationDetail: View {
     private var backgroundImage: some View {
         ZStack {
             GeometryReader { geometry in
+                let scrollOffset = geometry.frame(in: .global).minY
+                let headerOffset = scrollOffset > 0 ? -scrollOffset : 0
+                let headerHeight = scrollOffset > 0 ? geometry.size.height + scrollOffset : geometry.size.height
+                
                 if let url = publication.backgroundImageUrl {
                     WebImage(url: url)
                         .resizable()
                         .grayBackground()
-                        .aspectRatio(contentMode: .fill)
-                        .frame(width: geometry.size.width, height: 140)
+                        .scaledToFill()
+                        .frame(width: geometry.size.width, height: headerHeight)
                         .clipped()
+                        .offset(x: 0, y: headerOffset)
                 } else {
                     Rectangle() // TODO: Custom image
-                        .frame(width: geometry.size.width, height: 140)
+                        .frame(width: geometry.size.width, height: headerHeight)
                         .foregroundColor(.blue)
+                        .offset(x: 0, y: headerOffset)
                 }
             }
+            .frame(height: 140)
+            
             HStack {
                 VStack(alignment: .leading) {
                     Button(action: {
                         self.presentationMode.wrappedValue.dismiss()
                     }) {
                         Image.volume.backArrow
+                            .foregroundColor(.white)
                             .padding(.top, 55)
                             .padding(.leading, 20)
+                            .shadow(color: .black, radius: 4, x: 0, y: 0)
```

Comment:
I think you can do `.shadow(color: .black, radius: 4)` 

x, y should be default to 0 according to [this](https://developer.apple.com/documentation/swiftui/menu/shadow(color:radius:x:y:))